content/300-guides/100-performance-and-optimization/090-metrics.mdx

```diff
@@ -0,0 +1,497 @@
+---
```

Comment:
Why was this page moved to the Guides section of the docs? It is the Concepts->Components documentation of the Metrics feature. The "Guides" section of the docs usually does not contain this (without also having at least an equivalent in Concepts=>Components)

Reply:
Although that is still inconsistent with all the other documentation we have, it is definitely better than not having a page there at all.

Reply:
This is probably reasonably moot, because we're re-orging the docs soon, but I'll switch the content of the two pages, so that the main content is in components, and we link to it from the guides section.